### DISCUSSION TITLE
Custom resolutions are not shown - 自定义分辨率还用不了

### DISCUSSION THREAD

**Discussion by shielofz** [nomi-san/parsec-vdd#14] [Q&A] [answered]
添加了自定义分辨率 选择之后没有反应

**Comment by nomi-san** [comment, marked as answer]
You need to remove the display and then add again to apply custom resolutions.

![image](https://github.com/nomi-san/parsec-vdd/assets/38210249/8a2ec79e-6f0a-408e-83a4-1ebf34e041e9)
![image](https://github.com/nomi-san/parsec-vdd/assets/38210249/73a8b4b0-a1ea-4fcd-abfe-4683febf8368)

**Comment by SLALALALLA** [comment]
I use my phone as a secondary screen, but no matter how I set the resolution, the image is always cropped and cannot be displayed fully.
